src/lightning_app/core/app.py

```diff
@@ -133,6 +136,10 @@ def __init__(
 
         logger.debug(f"ENV: {os.environ}")
 
+        # update index.html,
+        # this should happen once for all apps before the ui server starts running.
+        add_meta.update_index_file_with_info(FRONTEND_DIR, info=info)
```

Comment:
This means that the default `index.html` will be changed for all apps, is this intended?

Reply:
Imagine these two apps:
app_1.py
```python

app = L.LightningApp(info=AppInfo(name="Im a cool app"))

```
the title of the HTML will be "Im a cool app"

app_2.py
```python

app = L.LightningApp()

```

if you run app_1 first then app_2, will app_2 have "Im a cool app" title ?

Reply:
no, it'll be reverted to original.